pkg/proxy/iptables/proxier.go

```diff
@@ -159,7 +159,7 @@ type Proxier struct {
 	endpointSlicesSynced bool
 	servicesSynced       bool
 	needFullSync         bool
-	initialized          int32
+	initialized          atomic.Int32
```

Comment:
if we're switching from atomic ops to atomic types you can make this `atomic.Bool`, since that's how it's really used